README.md

```diff
@@ -144,6 +146,38 @@ A `RequirementType` of `All` changes the traversal. First, if there are no filte
 
 In the above example, `FeatureW` specifies a `RequirementType` of `All`, meaning all of its filters must evaluate to true for the feature to be enabled. In this case, the feature will be enabled for 50% of users during the specified time window.
 
+#### Microsoft Feature Flag Schema
+
+The feature management library also supports the usage of [`Microsoft Feature Flag schema`](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json) to declare feature flags.
+
+If you want to use the `Microsoft Feature Flag schema`, you will need to follow the `Microsoft Feature Management Configuration schema` to set up feature flags. The schema can be found [here](https://github.com/Azure/AppConfiguration/tree/main/docs/FeatureManagement). The feature flag declarations should be listed in the `feature_flags` array under the `feature_management` section, as shown in the example below:

```

Comment:
> If you want to use the `Microsoft Feature Flag schema`, you will need to follow the `Microsoft Feature Management Configuration schema`

I feel like we should only talk about the Microsoft Feature Management schema. The flag schema simply falls in line as a result.

Reply:
I also feel it's worth explaining why this exists since it may be odd to see it declared immediately after the .NET schema.

This schema is language agnostic in origin and is supported by all Microsoft feature management libraries.